apps/synthmark.cpp

```diff
@@ -62,6 +62,7 @@ void usage(const char *name) {
            kDefaultSeconds);
     printf("    -b{burstSize} frames read by virtual hardware at one time, default = %d\n",
            kDefaultFramesPerBurst);
+    printf("    -B{burstNum} initial number of bursts, default = %d\n", 1);
```

Comment:
usage should say specific to LatencyMark.
I can make these changes later.

Reply:
I agree, thanks!